tox.ini

```diff
@@ -38,7 +38,7 @@ deps =
     django{22,32,40,main}: Pillow
     django22: Django>=2.2,<2.3
     django32: Django>=3.2,<3.3
-    django40: Django>=4.0,<4.1
+django40: Django>=4.0,<4.1
```

Comment:
Why was this committed?

Reply:
@thedrow Can I bother you with a proper review, if you have time?